server/views/pages/recallType.njk

```diff
@@ -7,7 +7,7 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
 
-        <span class="govuk-caption-l">Book a recall for {{ person.fullName }}</span>
+        <span class="govuk-caption-l">Book a recall for {{ person.firstName }} {{ person.lastName }}</span>
```

Comment:
This could stay as fullName? I think it's in the search API payload

Reply:
fullName isn't in the search payload yet, think you've added it in your branch

Reply:
ah fair enough